selfdrive/loggerd/loggerd.cc

```diff
@@ -198,62 +148,34 @@ void encoder_thread(int cam_idx) {
     while (!do_exit) {
       VisionIpcBufExtra extra;
       VisionBuf* buf = vipc_client.recv(&extra);
-      if (buf == nullptr) {
-        continue;
-      }
-
-      //printf("logger latency to tsEof: %f\n", (double)(nanos_since_boot() - extra.timestamp_eof) / 1000000.0);
-
-      // all the rotation stuff
-      {
-        pthread_mutex_lock(&s.rotate_lock);
-        pthread_mutex_unlock(&s.rotate_lock);
-
-        // wait if camera pkt id is older than stream
-        rotate_state.waitLogThread();
-
-        if (do_exit) break;
+      if (buf == nullptr) continue;
 
-        // rotate the encoder if the logger is on a newer segment
-        if (rotate_state.should_rotate) {
-          LOGW("camera %d rotate encoder to %s", cam_idx, s.segment_path);
+      s.last_camera_seen_tms = millis_since_boot();
```

Comment:
gate this on cam_info.trigger_rotate?

Reply:
Sure. Don't think it matters much in practice.